### PR TITLE
[cpu] [Coverity] fix Uninitialized issue in node mvn

### PR DESCRIPTION
**Pull request by tiger100256-hu**
### Details:
 - *fix Uninitialized issue in node mvn*

### Tickets:
 - *CVS-107182*

### CODE REVIEW

File: src/plugins/intel_cpu/src/nodes/mvn.h [hunk 1]
```diff
@@ -105,7 +105,8 @@ class MVN : public Node {
 
     std::vector<const void*> postOpsDataPtrs;
 
-    MVNAttrs mvnAttrs;
+    MVNAttrs mvnAttrs = {mvn_planar, std::make_tuple(0u, 0u, 0u, 0u, 0u), false, false, false, 0.0f,
+        INSIDE_SQRT, InferenceEngine::Precision(), InferenceEngine::Precision()};
```
Comment (ilya-lavrenov) [inline src/plugins/intel_cpu/src/nodes/mvn.h:109]: may be this should be a part of default ctor?

**Comment by tiger100256-hu** [comment]
@maxnick  I get message from @yuxu42 that the second part is not a real issue, so I didn't look into it.

**Comment by maxnick** [comment]
> @maxnick I get message from @yuxu42 that the second part is not a real issue, so I didn't look into it.

Ok, I see. Thank you for clarification.

**Review by maxnick — APPROVED** [review]

**Comment by maxnick** [comment]
@tiger100256-hu , we need green CI to merge it.

**Pull request merged**